# Set your detection topic here
#detection_topic: "/object_detection/detections"
detection_topic: "/body_tracker_array/position"

# Set your camera topic here 
#camera_topic: "/camera/rgb/image_rect_color"
#camera_topic: "/object_detection/detections_image"
#camera_topic: "/cv_camera/image_raw"
camera_topic: "/camera/color/image"

# Set your output topic here
output_topic: "/face_recognizer/faces"

# Set your output topic for RGB Image here
output_topic_rgb: "/face_recognizer/faces/image"
